extensions/amp-dailymotion/0.1/amp-dailymotion.js

```diff
@@ -34,8 +77,21 @@ class AmpDailymotion extends AMP.BaseElement {
   */
   preconnectCallback(opt_onLayout) {
     this.preconnect.url('https://www.dailymotion.com', opt_onLayout);
-    // Host that Dailymotion uses to serve JS needed by player.
-    this.preconnect.url('https://static1.dmcdn.net', opt_onLayout);
+  }
+
+  /**
+   * @override
+   */
+  supportsPlatform() {
+    return true;
+  }
+
+  /** @override */
+  isInteractive() {
+    // YouTube videos are always interactive. There is no YouTube param that
```

Comment:
update comments, also let's make sure there actually isn't a dailymotion param that can completely hide the UI controls.